app/packages/core/src/components/utils/editor/theme.ts

```diff
@@ -0,0 +1,235 @@
+import { HighlightStyle } from '@codemirror/language';
+import { EditorView } from '@codemirror/view';
+import { tags } from '@lezer/highlight';
+import { darken, Theme } from '@mui/material';
+
+/**
+ * `createTheme` creates a new theme for our `CodeMirrorEditor` component based on the provided MUI theme.
+ */
+export const createTheme = (theme: Theme, minimal: boolean) => {
+  return EditorView.theme(
+    {
+      '&.cm-editor': {
+        '&.cm-focused': {
+          outline: 'none',
+          outline_fallback: 'none',
+        },
+
+        backgroundColor: minimal ? theme.palette.background.default : theme.palette.background.paper,
+        height: '100%',
```

Comment:
```suggestion
        height: '100%',
        paddingLeft: "4px",
```

![image](https://user-images.githubusercontent.com/60616628/229778476-daa93ba1-6dec-4033-9a22-6e2901c0ff93.png)